test/TesterInternal/EventStoreTests/GetEventStoreProviderTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Orleans;
+using Orleans.Providers;
+using Orleans.Runtime.Storage;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestExtensions;
+using Xunit.Abstractions;
+using Orleans.Storage;
+using Orleans.Runtime.Configuration;
+using Xunit;
+using Orleans.Runtime;
+using Orleans.EventSourcing;
+
+namespace Tester.EventStorageTests
+{
+    
+    [Collection(TestEnvironmentFixture.DefaultCollection)]
+    public class GetEventStoreProviderTests : CommonEventStoreTests
+    {
+        private readonly EventStorageProviderManager manager;
+        private readonly ITestOutputHelper output;
+        private readonly TestEnvironmentFixture fixture;
+        private readonly IEventStorage eventStorage;
+
+       
+         public GetEventStoreProviderTests(ITestOutputHelper output, TestEnvironmentFixture fixture)
+        {
+            this.output = output;
+            this.fixture = fixture;
+
+            manager = new EventStorageProviderManager(
+                fixture.GrainFactory,
+                null,
+                new ClientProviderRuntime(fixture.InternalGrainFactory, null));
+            manager.LoadEmptyProviders().WaitWithThrow(TestConstants.InitTimeout);
+
+            var providerCfgProps = new Dictionary<string, string>();
+            var cfg = new ProviderConfiguration(providerCfgProps, null);
+            IEventStorageProvider provider = new GetEventStoreProvider();
+            provider.Init("GetEventStore", manager, cfg).WaitWithThrow(TestConstants.InitTimeout);
```

Comment:
does this always work or is it dependent on the environment? I assume it's the latter, so consider throwing a SkipException if it fails due to the environment not bein set up, and change all the [Fact] to [SkippableFact]